Dockerfile.rhel

```diff
@@ -49,6 +49,8 @@ COPY --from=build /build/jmx_prometheus_javaagent.jar ${PROMETHEUS_JMX_EXPORTER}
 
 WORKDIR $PRESTO_HOME
 RUN zip -q -d plugin/presto-elasticsearch/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
+RUN zip -q -d plugin/accumulo/log4j-1.2.17.jar org/apache/log4j/net/JMSAppender.class
+RUN zip -q -d plugin/phoenix/phoenix-client-4.14.1-HBase-1.4-embedded.jar org/apache/log4j/net/JMSAppender.class
```

Comment:
Just for our future selves: I pulled down the downstream 4.8 Presto image and exec-ed into locally, and ran the following command:

```bash
# WORKDIR $PRESTO_HOME
$ find ./ -name '*.jar' | xargs grep JMSAppender
...
```